src/components/Footer.jsx

```diff
@@ -1,7 +1,18 @@
 import React from "react";
-
+import { FaInstagram, FaDiscord } from "react-icons/fa";
+import { IoMail } from "react-icons/io5";
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="bg-gradient-to-r from-hiss-blue to-hiss-purple w-full p-4">
+      <div className="flex items-center font-poppins font-bold text-3xl text-white">
+        HISS
+        <div className="w-full max-h-full flex space-x-4 place-content-end">
+          {<FaInstagram className="text-2xl" />}{" "}
```

Comment:
things to add:
1. when hovered, make cursor pointer
2. when clicked, takes you to dummy links for now